libs/contrib/contrib.ipkg

```diff
@@ -89,6 +89,9 @@ modules = Control.ANSI,
 
           Data.Rel.Complement,
 
+          Data.Seq,
+          Data.Seq.Seq',
```

Comment:
Why are there 2 of these?

Reply:
The first one has its length in its type, the second not.